README.rst

```diff
@@ -1,4 +1,3 @@
-===========
```

Comment:
This line gave a different header level; after this change scrapy-poet is on a same level as e.g. Usage.

If that's a problem for docs, I think a little of duplication is fine (i.e. pull in changelog, but not readme).

Reply:
reverted